src/apps/subjects/constants.py

```diff
@@ -10,3 +10,11 @@ class SubjectStatus(str, Enum):
 class Relation(str, Enum):
     self = "self"
     admin = "admin"
+
+
+class SubjectTag(str, Enum):
+    CHILD = "Child"
+    TEACHER = "Teacher"
+    PARENT = "Parent"
+    GUARDIAN = "Guardian"
```

Comment:
As far as I've seen so far in Figma, Parent and Guardian are not 2 separate tags, but rather a single tag "Parent/Guardian". Do you have specs where it states these are two separate tags?

Reply:
FWIW, I have a PR in review where I named this tag `parent` but it's [translated to "Parent/Guardian"](https://github.com/ChildMindInstitute/mindlogger-admin/pull/1706/commits/67d50d8e7b4377a1874af0efd0660c18a8343667#diff-8b07a3883b40e08cd10b693229269699adc9846c6bb21b4f9e58b072c4b39fe6R865). Also I realize I picked lowercase for these values, but I can update my PR to make them uppercase as you have them here.

Reply:
I will not keep any list of possible tags in API, just put a todo comment to flag a future change